content/hardware/04.pro/boards/portenta-x8/tutorials/datalogging-iot/content.md

```diff
@@ -0,0 +1,416 @@
+---
+title: 'Data Logging with MQTT, Node-RED, InfluxDB and Grafana'
+description: 'This tutorial will show you how to set up a local data logging application using an MQTT broker, Node-RED, InfluxDB, Grafana, and the Arduino® Portenta X8.'
+difficulty: Intermediate
+tags:
+  - Docker
+  - MQTT
+  - Mosquitto
+  - Node-RED
+  - InfluxDB
+  - Grafana
+author: 'José Bagur, Taddy Chung'
+software:
+  - Terminal
+  - Docker
+hardware:
+  - hardware/04.pro/boards/portenta-x8
+---
+
+## Overview
+
+This tutorial will set up a typical Internet of Things (IoT) application: an MQTT data logger. For this application, we are going to use what we call in the Arduino team the "IoT-Quartet," four common building blocks that are popular and regularly used in these types of applications:
+
+- Mosquitto (MQTT broker) 
+- Node-RED
+- InfluxDB
+- Grafana
+
+These four blocks will be running locally on the [Arduino® Portenta X8](https://store.arduino.cc/products/portenta-x8) board. We will use data from an [Arduino® MKR WiFi 1010](https://store.arduino.cc/products/arduino-mkr-wifi-1010) board to test the data logging application.
+
+## Goals
+
+- Install, configure and run Mosquitto (MQTT broker) locally in the X8
+- Install, configure and run Node-RED locally in the X8
+- Install, configure and run InfluxDB locally in the X8
+- Install, configure and run Grafana locally in the X8
+- Send data from an Arduino® MKR WiFi 1010 board to the data logging application running locally in the X8

```

Comment:
Try to use the full name of the product "Arduino Portenta X8" instead of the friendly way (avoid possible confussions)

Reply:
Done!